docs/src/main/paradox/server-side/graceful-termination.md

```diff
@@ -0,0 +1,75 @@
+# Graceful termination
+
+## Graceful termination using `ServerTerminator`
+
+Akka HTTP provides two APIs to "stop" the server, either of them are available via the
+@java[@javadoc[ServerBinding](akka.http.javadsl.ServerBinding)]
+@scala[@scaladoc[ServerBinding](akka.http.scaladsl.Http$$ServerBinding)]
+obtained from starting the server (by using any of the `bind...` methods on the
+@java[@javadoc[Http](akka.http.javadsl.Http)]@scala[@scaladoc[Http](akka.http.scaladsl.HttpExt)] extension).
+
+The first method, called `unbind()` causes the server to *stop accepting new connections*, however any existing
+connections that are still being used will remain active.
```

Comment:
"until the client chooses to close them"?

Reply:
ok :)